### ISSUE TITLE
Shift to repair barriers?

### ISSUE THREAD

**Issue by TheTurkeyDev** [TheTurkeyDev/Call_Of_Minecraft-Zombies#97]
Should we use shifting as the mechanic to repair barriers instead of breaking a sign?

**Comment by Flowlessx** [comment]
yess :) is it possible to use a placeholder for your ingame points?

**Comment by TheTurkeyDev** [comment]
Currently no. What is listed under the placeholders wiki page is all there is

**Comment by Superfudge247** [comment]
This would be a great feature you could maybe play a title that indicates that you are repairing the barrier

**Comment by kilsnacks** [comment]
yes I would prefer shift

**Comment by bluhcops** [comment]
I think it should be the way it is so that players that don't have a shift function can still repair barriers.

**Comment by Superfudge247** [comment]
> I think it should be the way it is so that players that don't have a shift function can still repair barriers.

I disagree almost all platforms have the function to shift plus the plugin doesn't recognise the button being pressed it will just look for players crouching in a radius and all platforms have the ability to crouch
